inmemory.go

```diff
@@ -0,0 +1,82 @@
+package repohandler
+
+import (
+	"errors"
+
+	"github.com/google/uuid"
+)
+
+// InMemoryDB implements RepoHandler for a custom in memory database
+type InMemoryDB struct {
+	DB *map[string]interface{}
+}
+
+// Create inserts data by creating a generated uuid to track data
+func (in *InMemoryDB) Create(data interface{}) error {
+	uniqueID := uuid.New().String()
+	database := *in.DB
+	database[uniqueID] = data
+
+	return nil
+}
+
+// Get retrieves all data from the map
+func (in *InMemoryDB) Get() ([]interface{}, error) {
```

Comment:
`in *InMemoryDB` not a good name.

options are (sorted from best to good)

- `inMemoryDB`
- `imd`
- `receiver` when keeping things general and possibility of interchange of method between data structures.